js/gloss-transcription.js

```diff
@@ -862,19 +881,32 @@ function unmarkTPENLineElements(startEl, stopEl){
             for(const mark of nextEl.querySelectorAll(".pre-select")){
                 remark_map[nextEl.getAttribute("tpen-line-id")].push(mark.textContent)
             }
+            for(const mark of nextEl.querySelectorAll(".persists")){
+                persistent_map[nextEl.getAttribute("tpen-line-id")].push(mark.textContent)
+            }
             unmarkup = new Mark(nextEl)
             unmarkup.unmark({"className" : "pre-select"})
+            unmarkup.unmark({"className" : "persists"})
         }
         // Upstream from this the selection in stopEl is checked for a <mark>.  We know it does not have a <mark> here.
         remark_map[stopEl.getAttribute("tpen-line-id")] = []
         for(const mark of stopEl.querySelectorAll(".pre-select")){
             // For each thing you want to unmark, grab the text so we can remark it
             remark_map[stopEl.getAttribute("tpen-line-id")].push(mark.textContent)
         }
+        for(const mark of stopEl.querySelectorAll(".persists")){
+            // For each thing you want to unmark, grab the text so we can remark it
+            persistent_map[stopEl.getAttribute("tpen-line-id")].push(mark.textContent)
+        }
         unmarkup = new Mark(stopEl)
         unmarkup.unmark({"className" : "pre-select"})
+        unmarkup.unmark({"className" : "persists"})
+    }
+    const o = {
+        "pre-select" : remark_map,
+        "persists" : persistent_map
     }
-    return remark_map
+    return o
```

Comment:
This could probably just get returned instead of assigned for a second, but that's nitpicking.